integration/src/test/kotlin/kotlinx/benchmark/integration/InvalidTargetingTest.kt

```diff
@@ -0,0 +1,50 @@
+package kotlinx.benchmark.integration
+
+import kotlin.test.*
+
+class InvalidTargetingTest : GradleTest() {
+
+    @Test
+    fun testInvalidTargetForKotlinWASM() {
+        val runner = project("kotlin-multiplatform") {
+            configuration("invalidWasm") {
+                iterations = 1
+                iterationTime = 100
+                iterationTimeUnit = "ms"
+                reportFormat = "json"
+            }
+            register("wasm")
+            kotlin {
+                wasm("wasm") {
```

Comment:
Recently, we decided to use "wasmJs" name for wasm target. See https://github.com/Kotlin/kotlinx-benchmark/pull/129

Reply:
Noted.